### ISSUE TITLE
Language detection fails renders no element

### ISSUE THREAD

**Issue by kand617** [conorhastings/react-syntax-highlighter#72]
When the following element is rendered, I get a empty code element.

```objc
    Contacts* contacts = [[Contacts alloc]init];

```
![image](https://cloud.githubusercontent.com/assets/5092518/25770159/36415f54-3282-11e7-8ec3-f9c357f72d0a.png)

![image](https://cloud.githubusercontent.com/assets/5092518/25770156/2a9d57f2-3282-11e7-82b4-61b244758eb5.png)

**Comment by conorhastings** [comment]
@kand617 i was unable to reproduce, can you post a sample of your code? Here's what i tried

<img width="746" alt="screen shot 2017-05-07 at 7 11 46 am" src="https://cloud.githubusercontent.com/assets/8263298/25780362/905c3960-32f4-11e7-8aed-338572259411.png">
<img width="596" alt="screen shot 2017-05-07 at 7 11 38 am" src="https://cloud.githubusercontent.com/assets/8263298/25780364/91ffcd18-32f4-11e7-935b-cd3c2332a24e.png">

**Comment by kand617** [comment]
Hi @conorhastings ,

Perhaps I forgot to mention that it I am not using the whole suite of languages. Using the light version.
```

// // Import relavent languages
import SyntaxHighlighter, { registerLanguage } from 'react-syntax-highlighter/dist/light';
import js from 'highlight.js/lib/languages/javascript';
import objc from 'highlight.js/lib/languages/objectivec';
import php from 'highlight.js/lib/languages/php';
import xml from 'highlight.js/lib/languages/xml';
import ruby from 'highlight.js/lib/languages/ruby';
import cs from 'highlight.js/lib/languages/cs';
import python from 'highlight.js/lib/languages/python';
import java from 'highlight.js/lib/languages/java';
import go from 'highlight.js/lib/languages/go';

registerLanguage('javascript', js);
registerLanguage('js', js);
registerLanguage('objc', objc);
registerLanguage('objectivec', objc);

registerLanguage('php', php);
registerLanguage('html', xml);
registerLanguage('xml', xml);
registerLanguage('ruby', ruby);
registerLanguage('csharp', cs);
registerLanguage('cs', cs);

registerLanguage('python',python);
registerLanguage('java',java);
registerLanguage('go',go);

<SyntaxHighlighter> 
    {`Contacts* contacts = [[Contacts alloc]init];`}
 </SyntaxHighlighter>

**Comment by conorhastings** [comment]
@kand617 ahh, I think this is because the snippet is too small for highlightAuto to find a suitable language from those that are registered, I've published `v5.5.2` to make sure it at least renders the passed in string even if it's without highlighting
